bob-backend/src/pearl/core.rs

```diff
@@ -165,10 +176,11 @@ impl BackendStorage for Pearl {
             .disk_controllers
             .iter()
             .find(|dc| dc.can_process_operation(&op));
-        if let Some(disk_controller) = dc_option {
-            disk_controller
-                .put(op, key, data)
-                .await
+        if let Some(disk_controller) = dc_option.cloned() {
+            let data = data.clone();
+            spawn_call(async move {
```

Comment:
Reminder for the future (no need to do it right now):
to avoid DDOS by writing large records with short timeout, it would be good to limit the number of spawning requests with the help of Semaphore